server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java

```diff
@@ -2651,7 +2651,8 @@ private boolean hasDecidedQueryType() {
         || queryOp instanceof FillQueryOperator
         || queryOp instanceof LastQueryOperator
         || queryOp instanceof AggregationQueryOperator
-        || queryOp instanceof UDFQueryOperator;
+        || queryOp instanceof UDFQueryOperator
```

Comment:
```suggestion
        || queryOp instanceof UDTFQueryOperator
```

We can rename `UDFQueryOperator` to `UDTFQueryOperator`, which is clearer.